index.js

```diff
@@ -3,53 +3,433 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      quantity: 0,
+      type: "Veg"
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.50,
+      quantity: 0,
+      type: "Veg"
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.65,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.15,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 1.35,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.10,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.10,
+      quantity: 0,
+      type: "Fake Fruit"
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.05,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.50,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
+      price: 0.40,
+      quantity: 0,
+      type: "Fake Fruit"
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+const store = document.querySelector('.store--item-list')
+const total = document.querySelector('.total-number')
+const options = document.querySelector('#fruit')
+const order = document.querySelector('#order')
+
+order.addEventListener('input', () => {
+  if (order.value === "a_z") {
+    aToZ()
+      if (options.value === "All") {
+    storeListRender()
+  }
+  if (options.value === "Fruit") {
+    storeListFruitRender()
+  }
+  if (options.value === "Veg") {
+    storeListVegRender()
+  }
+  if (options.value === "Fake") {
+    storeListFakeRender()
+  }
+  }
+  if (order.value === "z_a") {
```

Comment:
This is a huge If statement block for an event listener. I'm unsure if it really needs to be that long or if there could be an easier way of handling things. Maybe you could call a new function with "options.value" and let that function return whatever you need to the event listener.